genfragments/ThirteenTeV/QstarToQW_Wmass/QstarToQW_M_2000_mW_170_TuneCP2_13TeV_pythia8_cfi.py

```diff
@@ -0,0 +1,38 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunes2017.PythiaCP2Settings_cfi import *
+from Configuration.Generator.PSweightsPythia.PythiaPSweightsSettings_cfi import *
+
+generator = cms.EDFilter("Pythia8GeneratorFilter",
+	comEnergy = cms.double(13000.0),
+	crossSection = cms.untracked.double(1),
```

Comment:
You might want to remove this line,
 	crossSection = cms.untracked.double(1),
The value won't be true and this will give you an error when it's submitted to the central cores for production since it checks whether the one you wrote in the fragment and the result matches or not